README.md

```diff
@@ -11,7 +11,7 @@ Docker powered web development environments for macOS, Windows, and Linux. Unifi
 - Full power of Docker Compose to suit any project without artificial syntax limitations
 - Unprecedented ability to automate routine tasks, slashing the time to on-board new team members.
 
-    Create your [custom automation](https://docs.docksal.io/fin/custom-commands/) in Bash, PHP or Node
```

Comment:
nope

Reply:
Ah, I see.

Reply:
Could you roll in back?

Reply:
I don't understand what you're asking me to do. I added the spaces back and the only difference in this line now is the extra comma that I added.

Reply:
Oh I see, thanks